src/bin/bat/config.rs

```diff
@@ -16,8 +16,22 @@ pub fn config_file() -> PathBuf {
 }
 
 pub fn get_args_from_config_file() -> Result<Vec<OsString>, shell_words::ParseError> {
+    let mut config_file_path = vec![PathBuf::from("/etc/bat/config")];
+    config_file_path.push(default_config_path());
+
+    let mut config_args = vec![];
+
+    for path in config_file_path.iter() {
+        let args = read_args_from_config_file(path.to_path_buf());
```

Comment:
I think you could avoid the `.to_path_buf()` if `read_args_from_config_file` simply takes a `&Path`.